configs/components/echam/echam.yaml

```diff
@@ -382,26 +382,41 @@ choose_scenario:
                         swflux: histswflux
                         MAC-SP: MAC-SP
 
-        # Shared Socioeconomic Pathways (SSPs) scenarios. Forcing data are given
-        # under the choose_scenario_type block
-        ssp585:
-                scenario_type: cmip6
-
-        ssp370:
-                scenario_type: cmip6
-
-        ssp126:
-                scenario_type: cmip6
-
-        ssp245:
-                scenario_type: cmip6
-
-        ssp534os:
-                scenario_type: cmip6
-
         cmip6hist:
                 scenario_type: cmip6
```

Comment:
This syntax annoys me. We should have some way of doing an "or", but that requires touching the parser, which I do not want to do, unless I get to blow it up completely and re-write it from scatch.